app/assets/javascripts/reducers/columns/done.js

```diff
@@ -4,15 +4,26 @@ const initialState = {
   stories: [],
 };
 
+const orderByAcceptedAt = (stories) => {
+  const orderedItems = [...stories];
+
+  orderedItems.sort(function(a,b){
+    return new Date(a.acceptedAt) - new Date(b.acceptedAt);
```

Comment:
Same here about extracting the function. Also, since they're in a comparable string format, we could avoid this `new Date`  here and compare the strings and have the same result. Can we try that?